pxr/imaging/hgiMetal/CMakeLists.txt

```diff
@@ -8,7 +8,11 @@ if (NOT ${PXR_BUILD_GPU_SUPPORT} OR NOT ${PXR_ENABLE_METAL_SUPPORT})
 endif()
 
 FIND_LIBRARY(METAL_LIBRARY Metal)
-FIND_LIBRARY(APPUIKIT_LIBRARY AppKit)
+if (APPLEIOS)
```

Comment:
should we pop these FIND_LIBRARIES all the way out to Packages, so they are universally available across the build, rather than repeating the check here & there?

Reply:
I think that's a good idea - I'll put that in place

Reply:
They've been moved and wiped